### DISCUSSION TITLE
Estou sem palavras pelo que está acontecendo

### DISCUSSION THREAD

**Discussion by luis007007** [community/community#132479] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Tem uma pessoa conhecida, e eu estou sendo prejudicado com o trabalho queria saber muito o nick dessa conhecida e tirar o 4 repositórios

**Comment by Hritickjha** [comment]
Response #1

**Reply by Hritickjha** [reply]
Answer

**Comment by LiteBrite82** [comment]
Hi @luis007007 , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
